modules/tooling-support/src/main/java/org/mule/runtime/module/tooling/internal/DefaultToolingService.java

```diff
@@ -44,11 +48,16 @@ public ConnectivityTestingServiceBuilder newConnectivityTestingServiceBuilder()
    */
   @Override
   public Application createApplication(File applicationLocation) throws IOException {
-    Application application = applicationFactory.createArtifact(applicationLocation, empty());
+    Application application = applicationFactory.createArtifact(applicationLocation, ofNullable(createApplicationProperties()));
     application.install();
     application.lazyInit();
     application.start();
     return application;
   }
 
+  private Properties createApplicationProperties() {
+    Properties properties = new Properties();
+    properties.setProperty(MULE_MUTE_APP_LOGS_DEPLOYMENT_PROPERTY, "true");
+    return properties;
```

Comment:
Confirm with Fabian that properties are not going to be modified so we can create this only once and reuse the same instance for every createApplication invocation.

Reply:
They are not meant to be modified right now, but could be on the future, so I wouldn't do this